woof-code/rootfs-packages/ptheme/usr/sbin/diricons.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/ash
+# called from ptheme_gtk
+# link special theme icons to */.DirIcon
+
+[ -e "$HOME/.config/gtk-3.0/settings.ini" ] && \
+	ICON_THEME="$(grep "gtk-icon-theme-name" $HOME/.config/gtk-3.0/settings.ini)" \
+	|| exit
+ICON_THEME="${ICON_THEME##*=}"
+ICON_THEME="${ICON_THEME/ /}"
+
+# link
+ln_diricon() {
+	EPATH=$1
+	ICON=$2
+	[ -e "$ICON" ] && \
+		(cd "$EPATH"; ln -svf "$ICON" ./.DirIcon )
+}
+
+for icon in $(ls "/usr/share/icons/$ICON_THEME/48/places/" | grep "^directory");do
+	if echo $icon|grep -q 'directory-home' ;then
+			ln_diricon "$HOME" "/usr/share/icons/$ICON_THEME/48/places/$icon"
```

Comment:
Is it possible to link to the .svg and let ROX-Filer resize to 48 when it renders the icon?

Reply:
All icons _are_ svg in the native puppy themes.